subsys/bootloader/Kconfig

```diff
@@ -121,15 +122,22 @@ menuconfig IS_SECURE_BOOTLOADER
 	bool "Current app is bootloader"
 	select SECURE_BOOT_VALIDATION
 	select SW_VECTOR_RELAY if SOC_SERIES_NRF51X
+	depends on NRFX_NVMC
 	help
 	  This option is set by the first stage bootloader app to include all
 	  files and set all the options required.
 
 if IS_SECURE_BOOTLOADER
 
-partition=B0_IMAGE
-partition-size=0x7000
-source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
+config PM_PARTITION_SIZE_PROVISION
+	hex
+	default 0x0 if SOC_NRF9160
+	default 0x1000 if !SOC_NRF9160
+
+config PM_PARTITION_SIZE_B0_IMAGE
+	hex
```

Comment:
Shouldn't we have a prompt here? At least for non SPU based devices it makes sense to me.

Reply:
Added the prompt from the template config. For PROVISION_SIZE, the prompt is only for non-91 devices.